draft-ietf-quic-recovery.md

```diff
@@ -1150,6 +1150,9 @@ This document has no IANA actions.
 We now describe an example implementation of the loss detection mechanisms
 described in {{loss-detection}}.
 
+The pseudocode snippets in this section are licensed as Code Components; see the
+copyright notice above.
```

Comment:
```suggestion
copyright notice.
```

I get that we can't cite this directly, but I think we're OK not saying "above".